### DISCUSSION TITLE
保持唤醒和超时重启应该怎么设置？

### DISCUSSION THREAD

**Discussion by wyfang** [constanline/XQuickEnergy#440] [Q&A/问答时间]
我忘记了默认是什么，但是我是都打开，但是有个问题就是经常玩手机会自动跳转到支付宝app。这个跳转很打扰正常使用手机，所以是否有办法不会跳转，又能持续保持功能？

**Comment by Zhangfanyu1222** [comment]
把“超时时重启Activity”关了就不会跳转支付宝了

**Reply by wyfang** [reply]
那是不是就有概率不会偷能量了？

**Reply by Zhangfanyu1222** [reply]
偷能量那段时间保活做好就行啦

**Comment by zhudongdong1105** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
